test/hw4/test_task3.py

```diff
@@ -0,0 +1,28 @@
+from All_home_works.hw4.task3 import my_precious_logger
+
+ERROR = "error: 404"
+TEXT = "OK: 200"
+
+
+def test_my_precious_logger__error_msg__test_msg_in_stderr(capsys):
```

Comment:
It's ok to join this test with test_my_precious_logger__error_msg__test_none_in_stdout